Editor/Scripts/DCCIntegration/Launchers/BlenderLauncher.cs

```diff
@@ -0,0 +1,244 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.MeshSync.Editor
+{
+    internal class BlenderLauncher : IDCCLauncher
+    {
+        const string menuItem_OpenInBlender = "Assets/MeshSync/Open in blender";
+
+        const string editorSettingPath = "MESHSYNC_BLENDER_PATH";
+
+        System.Diagnostics.Process m_blenderProcess;
+        static bool redirectBlenderToUnityConsole;
+
+        public RunMode runMode { get; set; }
+
+        ~BlenderLauncher()
+        {
+            Cleanup();
+        }
```

Comment:
I made it IDisposable

Reply:
The full implementation of the pattern requires more changes when Finalize is involved https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose
There are also some caveats when using finalizers https://pvs-studio.com/en/blog/posts/csharp/0437/ 

From my point of view, either there is a good reason to use them or its a code smell on ownership and lifetime management of objects

Reply:
Seems like the conversation here got stopped ? @seandillon92 @schinkowski 

I tried to verify the content of `Dispose()`, and it seems that it is used to to kill Blender which was previously launched by Unity.
If this is the case, then I'd actually think it's friendlier for users to let them shutdown Blender by themselves, as they might have edited the blend file, and haven't saved it, for example.

Reply:
I've removed it from the destructor and no longer kill the blender process now. I thought it would be more user friendly to not have to close blender and avoid having multiple processes open that can interfere with each other but don't mind either way.

Reply:
Actually, I've changed it so we stilll force close blender when it was running in background mode, otherwise the user would need to force close it through task manager.

Reply:
Ok. I am merging this PR.
The `Dispose()`, at least in this case, seems good to me, but please let us know if you have any concerns @seandillon92

Reply:
Looks good to me